hack/local-up-cluster.sh

```diff
@@ -378,11 +378,15 @@ cleanup()
 
   # Check if the proxy is still running
   [[ -n "${PROXY_PID-}" ]] && PROXY_PIDS=$(pgrep -P ${PROXY_PID} ; ps -o pid= -p ${PROXY_PID})
-  [[ -n "${PROXY_PIDS-}" ]] && sudo kill ${PROXY_PIDS}
+  if [[ -n "${PROXY_PIDS-}" ]]; then
+    rm -f /tmp/kube-proxy.yaml && sudo kill ${PROXY_PIDS}
```

Comment:
Current code would work but it has some drawbacks:
- if `rm` returns 1, the script will never kill the processes
- if proxy server has failed to start, `PROXY_PIDS` will be empty and we leave the config file

I'd suggest to just add `rm -f /tmp/kube-proxy.yaml || :` after killing the processes.

Reply:
Good catch